examples/CollectionViewWithViewControllerCells/Sample/ViewController.m

```diff
@@ -90,16 +92,17 @@ - (void)reloadTapped
 #pragma mark -
 #pragma mark ASCollectionView data source.
 
-- (ASCellNode *)collectionView:(ASCollectionView *)collectionView nodeForItemAtIndexPath:(NSIndexPath *)indexPath
+- (ASCellNodeBlock)collectionView:(ASCollectionView *)collectionView nodeBlockForItemAtIndexPath:(NSIndexPath *)indexPath
 {
-  ASCellNode *node = [[ASCellNode alloc] initWithViewControllerBlock:^UIViewController *{
-    return [[ImageViewController alloc] initWithImage:_sections[indexPath.section][indexPath.item]];
-  } didLoadBlock:nil];
-  
-  node.layer.borderWidth = 1.0;
-  node.layer.borderColor = [UIColor blackColor].CGColor;
-  
-  return node;
+  UIImage *image = _sections[indexPath.section][indexPath.item];
+  return ^{
+    return [[ASCellNode alloc] initWithViewControllerBlock:^UIViewController *{
+      return [[ImageViewController alloc] initWithImage:image];
+    } didLoadBlock:^(ASDisplayNode * _Nonnull node) {
+      node.layer.borderWidth = 1.0;
```

Comment:
These properties are very inefficient - I believe they cause offscreen rendering (and should be highlighted in yellow).  We should eliminate this code soon even if we don't have an immediate replacement, maybe just remove the border entirely for now.